runsc/boot/network.go

```diff
@@ -69,6 +70,8 @@ var (
 // Network exposes methods that can be used to configure a network stack.
 type Network struct {
 	Stack *stack.Stack
+
+	ExternalStack externalstack.ExternalStack
```

Comment:
Comment to explain what this is and how it's used. Maybe something like
```
ExternalStack is an optional network stack to use in place of netstack when non-nil.
```

Reply:
Thanks for bringing this up. Updated.